sys/include/net/nanocoap.h

```diff
@@ -434,7 +494,32 @@ unsigned coap_get_content_type(coap_pkt_t *pkt);
  * @returns     -ENOSPC     if URI option is larger than NANOCOAP_URI_MAX
  * @returns     nr of bytes written to @p target (including '\0')
  */
-int coap_get_uri(coap_pkt_t *pkt, uint8_t *target);
+static inline int coap_get_uri(const coap_pkt_t *pkt, uint8_t *target)
```

Comment:
If @kaspar030 [comment](https://github.com/RIOT-OS/RIOT/pull/8920/files#r180752431) is applied, the return type also propagates to this function.